gdx/src/com/badlogic/gdx/maps/tiled/BaseTmjMapLoader.java

```diff
@@ -0,0 +1,746 @@
+		}
+		if (staggerIndex != null) {
+			mapProperties.put("staggerindex", staggerIndex);
+		}
+		if (mapBackgroundColor != null) {
+			mapProperties.put("backgroundcolor", mapBackgroundColor);
+		}
+		this.mapTileWidth = tileWidth;
+		this.mapTileHeight = tileHeight;
+		this.mapWidthInPixels = mapWidth * tileWidth;
+		this.mapHeightInPixels = mapHeight * tileHeight;
+
+		if (mapOrientation != null) {
+			if ("staggered".equals(mapOrientation)) {
+				if (mapHeight > 1) {
+					this.mapWidthInPixels += tileWidth / 2;
+					this.mapHeightInPixels = mapHeightInPixels / 2 + tileHeight / 2;
+				}
+			}
+		}
+
+		JsonValue properties = root.get("properties");
+		if (properties != null) {
+			loadProperties(map.getProperties(), properties);
+		}
+
+		JsonValue tileSets = root.get("tilesets");
+		for (JsonValue element : tileSets) {
+			loadTileSet(element, tmjFile, imageResolver);
+
+		}
+		JsonValue layers = root.get("layers");
+
+		for (JsonValue element : layers) {
+			loadLayer(map, map.getLayers(), element, tmjFile, imageResolver);
+		}
+
+		// update hierarchical parallax scrolling factors
+		// in Tiled the final parallax scrolling factor of a layer is the multiplication of its factor with all its parents
+		// 1) get top level groups
+		final Array<MapGroupLayer> groups = map.getLayers().getByType(MapGroupLayer.class);
+		while (groups.notEmpty()) {
+			final MapGroupLayer group = groups.first();
+			groups.removeIndex(0);
+
+			for (MapLayer child : group.getLayers()) {
+				child.setParallaxX(child.getParallaxX() * group.getParallaxX());
+				child.setParallaxY(child.getParallaxY() * group.getParallaxY());
+				if (child instanceof MapGroupLayer) {
+					// 2) handle any child groups
+					groups.add((MapGroupLayer)child);
+				}
+			}
+		}
+
+		for (Runnable runnable : runOnEndOfLoadTiled) {
+			runnable.run();
+		}
+		runOnEndOfLoadTiled = null;
+
+		return map;
+
+	}
+
+	protected void loadLayer (TiledMap map, MapLayers parentLayers, JsonValue element, FileHandle tmjFile,
+		ImageResolver imageResolver) {
+		String type = element.getString("type", "");
+		switch (type) {
+		case "group":
+			loadLayerGroup(map, parentLayers, element, tmjFile, imageResolver);
+			break;
+		case "tilelayer":
+			loadTileLayer(map, parentLayers, element);
+			break;
+		case "objectgroup":
+			loadObjectGroup(map, parentLayers, element);
+			break;
+		case "imagelayer":
+			loadImageLayer(map, parentLayers, element, tmjFile, imageResolver);
+			break;
+		}
+	}
+
+	protected void loadLayerGroup (TiledMap map, MapLayers parentLayers, JsonValue element, FileHandle tmjFile,
+		ImageResolver imageResolver) {
+		if (element.getString("type", "").equals("group")) {
+			MapGroupLayer groupLayer = new MapGroupLayer();
+			loadBasicLayerInfo(groupLayer, element);
+
+			JsonValue properties = element.get("properties");
+			if (properties != null) {
+				loadProperties(groupLayer.getProperties(), properties);
+			}
+			for (JsonValue child : element) {
```

Comment:
The loadLayerGroup was parsing incorrectly leading to grouped layers being ignored entirly. You can do this instead to fix it.
```
JsonValue layers = element.get("layers");
				if (layers != null) {
					 for (JsonValue child : layers) {
						  loadLayer(map, groupLayer.getLayers(), child, tmjFile, imageResolver);
					 }
				}
```

For the sake of completion here is the newly modified method in it's entirety:
```
protected void loadLayerGroup (TiledMap map, MapLayers parentLayers, JsonValue element, FileHandle tmjFile,
		 ImageResolver imageResolver) {
		  if (element.getString("type", "").equals("group")) {
				MapGroupLayer groupLayer = new MapGroupLayer();
				loadBasicLayerInfo(groupLayer, element);

				JsonValue properties = element.get("properties");
				if (properties != null) {
					 loadProperties(groupLayer.getProperties(), properties);
				}

				JsonValue layers = element.get("layers");
				if (layers != null) {
					 for (JsonValue child : layers) {
						  loadLayer(map, groupLayer.getLayers(), child, tmjFile, imageResolver);
					 }
				}

				for (MapLayer layer : groupLayer.getLayers()) {
					 layer.setParent(groupLayer);
				}

				parentLayers.add(groupLayer);
		  }
	 }
```